File: my_vaccine_app/lib/screens/mother/motherUser.dart
class MotherUser {
  String password;
  String phone;
  String username;
  String identityNumber;
  String device_token;
  MotherUser({
    required this.password,
    required this.phone,
    required this.username,
    required this.identityNumber,
    required this.device_token,
  });

  Map<String, dynamic> toJson() {
    return {
      'password': password,
      'phone': phone,
      'username': username,
      'identity_number': identityNumber,
      'device_token': device_token,
    };
  }
}

File: newborn-app/app/Models/motherUser.php
<?php

namespace App\Models;

use Laravel\Sanctum\HasApiTokens;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Notifications\Notifiable;

class MotherUser extends Model
{
    use HasApiTokens, HasFactory, Notifiable;

    protected $table = 'mother_users';
    protected $fillable = ['password', 'phone', 'username', 'identity_number','device_token'];

    public function role()
    {
        return $this->belongsTo(Role::class);
    }
    public function mother()
    {
        return $this->belongsTo(Mother::class, 'identity_number', 'identity_number');
    }
}
